pandas/_libs/lib.pyx

```diff
@@ -3038,3 +3038,23 @@ def is_bool_list(obj: list) -> bool:
 
     # Note: we return True for empty list
     return True
+
+
+def dtypes_all_equal(list types):
```

Comment:
can use python-style annotation with bool return

Reply:
It seems you can't combine that with a `not None`, so left as is, but I added it to the pyi file alongside all others functions in lib.pyx